src/components/CategoryFilter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import categoryOptions from '../containers/constants';
+
+function CategoryFilter({ category, handleCategory }) {
```

Comment:
### [Optional]
we can achieve the same result with less code in a more reasable way

`const CategoryFilter = ({ category, handleCategory }) => {`